src/ui/ResultsFiltersPreferences/ResultsFiltersPreferences.ts

```diff
@@ -258,11 +258,16 @@ export class ResultsFiltersPreferences extends Component {
 
   private buildAdvancedFilters() {
     this.advancedFilters = $$('div', { className: 'coveo-advanced-filters' }, l('Create')).el;
+    new AccessibleButton()
+      .withElement(this.advancedFilters)
+      .withLabel(l('Create'))
```

Comment:
Since the button already contains text, a label would be redundant, you can use `withoutLabelOrTitle`.

Reply:
Ahh!